packages/flutter_tools/test/commands.shard/permeable/packages_test.dart

```diff
@@ -243,9 +241,9 @@ void main() {
       removeGeneratedFiles(projectPath);
 
       final PackagesCommand command = await runCommandIn(projectPath, 'get');
-      final PackagesGetCommand getCommand = command.subcommands['get'] as PackagesGetCommand;
+      final PackagesGetCommand? getCommand = command.subcommands['get'] as PackagesGetCommand?;
```

Comment:
Here (and below) can you cast when initializing `getCommand` so you don't need the bang when referencing it?
```suggestion
      final PackagesGetCommand getCommand = command.subcommands['get']! as PackagesGetCommand;
```